internal/tfimport/resources/google_container_cluster.go

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package resources
+
+import (
+	"fmt"
+
+	"github.com/GoogleCloudPlatform/healthcare-data-protection-suite/internal/terraform"
+)
+
+// GkeClusterImporter defines a struct with the necessary information for a GKE cluster to be imported.
+type GkeClusterImporter struct{}
+
+// ImportID returns the GCP project and bucket name for use in importing.
+func (b *GkeClusterImporter) ImportID(rc terraform.ResourceChange, pcv ProviderConfigMap) (string, error) {
+	project := fromConfigValues("project", rc.Change.After, pcv)
+	if project == nil {
+		return "", fmt.Errorf("could not find project in resource change or provider config")
+	}
+
+	location := fromConfigValues("location", rc.Change.After, pcv)
+	if project == nil {
+		return "", fmt.Errorf("could not find location in resource change or provider config")
+	}
+
+	name := fromConfigValues("name", rc.Change.After, nil)
+	if name == nil {
+		return "", fmt.Errorf("could not find cluster name in resource change or provider config")
+	}
+
+	return fmt.Sprintf("projects/%v/locations/%v/clusters/%v", project, location, name), nil
+}
```

Comment:
p.s. isn't there an exists check missing?

Reply:
Not sure what you mean. If the cluster exists, it'll be imported. If it doesn't exist, it will be handled and surfaced as an appropriate log message.

Do you mean checking if the resource has already been imported? If it has, then it wouldn't show up in the plan as a "create" change.

Reply:
"If it doesn't exist, it will be handled and surfaced as an appropriate log message."

OK, was just making sure this case was handled. LGTM then

Reply:
Yes, it's here: https://github.com/GoogleCloudPlatform/healthcare-data-protection-suite/blob/master/cmd/tfimport/main.go#L134-L135